README.md

```diff
@@ -10,7 +10,7 @@
 	1. unzip LeesStore app
 	2. Unzip your app
 	3. Suppose you want to generate a new entity Fig
-	4. run "python generateEntityFromLeesStore.pl Fig Figs (read below for setting up this)
+	4. run "python generateEntityFromLeesStore.py Fig Figs (read below for setting up this)
```

Comment:
Not sure why these were all .pl when the underlying files were .py.  I updated the documentation, but this could easily be something I don't understand about Python in which case I can revert.